tests/arch/arm64/arm64_gicv3_its/src/main.c

```diff
@@ -100,8 +99,7 @@ static void test_gicv3_its_irq_simple(void)
 			"IRQ %d of DeviceID %x EventID %d failed",
 			irqn, device_id, event_id);
 }
-
```

Comment:
nit: keep the blank line here, it'll make the diff cleaner

Reply:
Fixed.

Reply:
GitHub seems to be having issue with the force push so it may not be immediately obvious this was fixed so please check.